govc/vm/disk/create.go

```diff
@@ -113,10 +115,31 @@ func (cmd *create) Run(ctx context.Context, f *flag.FlagSet) error {
 		return nil
 	}
 
+	backing := disk.Backing.(*types.VirtualDiskFlatVer2BackingInfo)
+
 	if cmd.Thick {
-		backing := disk.Backing.(*types.VirtualDiskFlatVer2BackingInfo)
 		backing.ThinProvisioned = types.NewBool(false)
-		backing.EagerlyScrub = types.NewBool(cmd.Eager)
+	}
+
+	backing.EagerlyScrub = types.NewBool(cmd.Eager)
+
+	switch cmd.DiskMode {
```

Comment:
I noticed "dependent" in the UI becomes persistent. I will remove dependent from the new string array